docs/modules/ROOT/pages/how-to-guides/testing_applications/enable_snyk_check_for_a_product.adoc

```diff
@@ -0,0 +1,44 @@
+= Enable snyk task for a product
+
+== Description:
+
+The sast-snyk-check task uses Snyk Code tool to perform Static Application Security Testing (SAST) for Snyk, a popular cloud-native application security platform.
+
+Snyk's SAST tool uses a combination of static analysis and machine learning techniques to scan an application's source code for potential security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.
+
+> NOTE: This task is executed only if the user provides a Snyk token stored in a secret in their namespace. The name of the secret then needs to be supplied in the `snyk-secret` pipeline parameter.
+
+== Steps to enable snyk:
+
+* Sign up/login at https://app.snyk.io/
+* Obtain Snyk token by navigating to the Account / General / Auth Token section
++
+image::account_settings.png[alt=Snyk account settings]
```

Comment:
delete all images. (see my note from yesterday about screenshots.)

Reply:
@sonam1412 would you mind putting this procedure into ADOC form and removing the images? i can more easily/quickly review this after that.

Reply:
this is already into a adoc format. Am I missing something ?